### PR TITLE
Bump the UI package to v2.39.0-rc.0

### PR DESCRIPTION
**Pull request by codesome**
Apparently the release process was slightly modified and I wasn't aware of it. Read the release file carefully now.

Also, looks like the CI is broken for the release https://github.com/prometheus/prometheus/actions/runs/3149723579/jobs/5122024911, promu not found. I will try to fix that in this PR as well.

### CODE REVIEW

**Comment by juliusv** [comment]
:+1: 

> promu not found. I will try to fix that in this PR as well.

Is that still coming in this PR?

**Comment by codesome** [comment]
> did you run the command `make bump-version` @codesome ? It looks like the `package.json` is not well updated.

I did `make ui-bump-version`. I will see what is the error.

> Is that still coming in this PR?

Yes, I will check today.

File: .github/actions/publish_release/action.yml [hunk 1]
```diff
@@ -27,11 +27,11 @@ runs:
...
         enable_docker_multibuild: true
     - uses: ./.github/actions/restore_artifacts
-    - run: promu crossbuild tarballs
+    - run: ~/go/bin/promu crossbuild tarballs
       shell: bash
-    - run: promu checksum .tarballs
+    - run: ~/go/bin/promu checksum .tarballs
       shell: bash
-    - run: promu release .tarballs
+    - run: ~/go/bin/promu release .tarballs
```
Comment (codesome) [inline .github/actions/publish_release/action.yml:34]: This attempted fix comes from [here](https://github.com/prometheus/prometheus/blob/8dbb2eaf0bafd3563dc7d295fefa47cbb8c2a36a/.github/actions/build/action.yml#L19) which seems to be working

Comment (codesome) [reply .github/actions/publish_release/action.yml:34]: We will know if this change works when I push another release tag for this PR

**Comment by codesome** [comment]
@Nexucis @juliusv should be good for another review. The CI error (hence the missing things from package-lock) was fixed by running `make`.

**Comment by juliusv** [comment]
Thanks, looks good to me :+1:

**Comment by codesome** [comment]
Ugh, promu problem fixed but it failed for other reason now https://github.com/prometheus/prometheus/actions/runs/3157067233/jobs/5138037261, and artifact are not being uploaded as well. Time to debug more.

File: web/ui/react-app/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@prometheus-io/app",
-  "version": "0.38.0",
+  "version": "0.39.0-rc.0",
```
Comment (Nexucis) [inline web/ui/react-app/package.json:3]: weirdly the script `bump-version` should have updated as well the dependencies in this package.json :O

**Pull request merged**